ci/code_checks.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+echo "inside $0"
```

Comment:
can you add some docs here on how to run things, maybe also in the contributing docs themselves.

Reply:
Yep, that makes a lot of sense, thank, didn't think about that before. Updated the PR